src/openprocurement/api/views/tender_document.py

```diff
@@ -24,6 +24,18 @@
             description="Tender related binary files (PDFs, etc.)")
 class TenderDocumentResource(APIResource):
 
+    def validate_award_document(self, operation):
+        if self.request.authenticated_role != 'auction' and self.request.validated['tender_status'] != 'active.enquiries' or \
+           self.request.authenticated_role == 'auction' and self.request.validated['tender_status'] not in ['active.auction', 'active.qualification']:
+            self.request.errors.add('body', 'data', 'Can\'t update document in current ({}) tender status'.format(self.request.validated['tender_status']))
+            self.request.errors.status = 403
+            return
+        if operation == 'update' and self.request.authenticated_role != self.context.author:
```

Comment:
`if (self.context.author and self.request.authenticated_role != self.context.author) or self.request.authenticated_role != self.tender_owner:` ??